src/api-v1.js

```diff
@@ -103,10 +106,17 @@ api.declare({
             url: url,
           });
         }
+        // TODO: Should only status === 'error' be considered a miss?  A file
+        // that's never been requested before could never be in the cache, so
+        // it not being there really is not an error case
+        this.monitor.count(`${service}.${region}.cache-miss`, 1);
```

Comment:
@jonasfj as it stands, anything statsums supports _should_ just work with tc-lib-monitor. I think this will work just fine [due to these lines](https://github.com/taskcluster/taskcluster-lib-monitor/blob/master/src/monitor.js#L43-L49). This is however, untested in tc-lib-monitor because I didn't want to duplicate the internal logic of statsum in the project. I think perhaps at some point we should expose that transformation you do and then I can use it in my mock. I don't think the current mock implementation will stacktrace over this usage, but it won't do anything particularly useful.